components/faqs/support-for-css-frameworks.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+
+const sampleTachyonsExample = (`
+const Button = styled.button.attrs({
+  className: 'pa4 ba b--black-80',
+})\`
+	background: red;
+	color: white;
+
+	font-size: 1em;
+	margin: 1em;
+	padding: 0.25em 1em;
+	border: 2px solid palevioletred;
+	border-radius: 3px;
+\`;
+
+/* And use in render as: */
+
+render(
+	<div>
+		<Button>Hello from Tachyons</Button>
+	</div>
+);
+`).trim()
+
+const CSSFrameworks = () => (
+  <SectionLayout title="Support for CSS Frameworks">
+    <p>Class names defined from a referenced CSS file can be used <b>along</b> with the style definitions! This will let styles to be separated from the markup making it very clean & readable.</p>
```

Comment:
This is a bit of a clunky intro, since we can be very specific:

> Integrating an existing CSS framework with styled-components is really easy! You can use its existing class-names alongside your components.

Please don't put in stuff like:

> This will let styles to be separated from the markup making it very clean & readable

Since we generally don't want to encourage everyone to use CSS frameworks and this can be somewhat of a fallacy depending on what you mean here, because markup is not related to the following intro.